Recrutify/Recrutify.DataAccess/Constants.cs

```diff
@@ -31,5 +31,10 @@ public static class InviteMessage
             public const string Subject = "Interview";
             public const int TimeZone = 3;
         }
+
```

Comment:
move ALL constants from this file to Constants file in Host project.
Why you duplicate Skype in both files??

Reply:
Why templates and so on are still in Data Access project ?